src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(100));
```

Comment:
magic number `100` should be `private static final` class level constant
again, we dont need to create ColorSupplier and Random each time, extract to the class-level fields:
![image](https://github.com/mate-academy/jv-lottery/assets/73041091/724938ed-c279-4a6d-8313-d857bca03053)